app/views/profiles/show.html.erb

```diff
@@ -1,15 +1,54 @@
+<%= stylesheet_link_tag 'profiles' %>
+
 <div class="mx-auto md:w-2/3 w-full flex">
   <div class="mx-auto">
     <% if notice.present? %>
       <p class="py-2 px-3 bg-green-50 mb-5 text-green-500 font-medium rounded-lg inline-block" id="notice"><%= notice %></p>
     <% end %>
 
-    <%= render @profile %>
+    <div class="profile-header-container">
```

Comment:
So instead of `class="profile-header-container"`, this would be approximately: `class="flex flex-row items-center gap-4 pb-2.5"`

`flex` = display: flex;
`flex-row` = flex-direction: row;
`items-center` = align-items: center;
`gap-4` = gap: 15%; (gap-4 is 16px, since there isn't a % equivalent in Tailwind)
`pb-2.5` = padding-bottom: 10px;

Reply:
If we _do_ want to make styles more easily re-usable for UI components, we can also define html tags and styles in Ruby helper modules. It might seem a little strange at first, but there are some real advantages to this -- such as testability! I can put together some examples to show you how it works.